### DISCUSSION TITLE
no images for Preset Toggle Buttons

### DISCUSSION THREAD

**Discussion by johans1234** [nguyenquyhy/Flight-Tracker-StreamDeck#140] [Q&A]
since a few minutes, all my images from the Preset Toggle Buttons are away and I am not possible to bring them back. Anyone else with this issue? I mean the images which I can set on the right hand side under "custom on/off images".

**Comment by rmroc451** [comment]
try killing the stream deck client with task manager, and restarting it, do they come back?

**Reply by johans1234** [reply]
sorry, was my mistake. Shame on me. My Flight Simulator was not started ;o((((

**Reply by rmroc451** [reply]
no problem @johans1234 !  Glad you got it sorted out!
